README.md

```diff
@@ -64,6 +45,4 @@ The backup strategy is **not a one-off activity**. It will require you to take s
 * Ensure that the git repo catalogs that you are "tracking" in the `${PERSONAL_CONFIGS_DIR}/repositories-*.yml` files are kept up-to-date so that resurrection in your new machine will be seamless
 * Ensure to run the `scripts/capture-defaults.sh` (with the export switch) to export and capture/backup your preferences for all installed applications from your current machine
 
-# Extras/Details
-
-Some utility scripts have been provided in this repo - which you can use to manage the backup strategy in a better fashion. Details can be found [here](Extras.md)
+# [Click here](GettingStarted-Basic.md) to get started with the basic setup
```

Comment:
i see your point about taking them through a journey/story-telling style. If so, then the table of contents on the top of the README might also need to be removed?